JudoPayDotNet/Models/PaymentModel.cs

```diff
@@ -144,6 +144,13 @@ protected PaymentModel()
 // ReSharper disable once UnusedMember.Global
         public JObject ClientDetails { get; set; }
 
+        /// <summary>
+        /// Details needed for MCC 6012 transactions
+        /// </summary>
+        [DataMember(EmitDefaultValue = false)]
+// ReSharper disable once UnusedMember.Global
+        public JObject PrimaryAccountDetails { get; set; }
```

Comment:
Adding this as a JObject won't it make hard for the library client to know which fields are mandatory and the right names?